# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kdeadmin/libtool-fix.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


A tiny "hack" for our /lib vs. /usr/lib situation. Too damn kde* uses an
own copy of libtool - we need to verify how the official GNU beast behaves
and maybe really fix that one.

  - Rene Rebe <rene@exactcode.de>

--- kdeadmin-3.1.4/admin/ltmain.sh.orig	2003-12-03 16:30:02.000000000 +0100
+++ kdeadmin-3.1.4/admin/ltmain.sh	2003-12-03 16:30:56.000000000 +0100
@@ -1963,9 +1963,12 @@
 	    dir="$ladir"
 	    absdir="$abs_ladir"
 	    libdir="$abs_ladir"
-	  else
+	  elif test -f "$libdir/$linklib" ; then
 	    dir="$libdir"
 	    absdir="$libdir"
+	  else
+	    dir="/lib"
+	    absdir="/lib"
 	  fi
 	else
 	  dir="$ladir/$objdir"
